scripts/src/seed-database.ts

```diff
@@ -0,0 +1,367 @@
+        },
+      ],
+      readyToEndChat: false,
+      isSilent: true,
+    },
+  },
+  {
+    kind: "takeSurvey",
+    name: "4. Post-chat survey",
+    config: {
+      questions: [
+        {
+          kind: "ScaleQuestion",
+          id: "3",
+          questionText:
+            "Rate the chat dicussion on a 1-10 scale.\nAlso indicate your overall feeling about the chat.",
+          lowerBound: "I did not enjoy the discussion at all (0/10)",
+          upperBound: "The dicussion was a perfect experience to me (10/10)",
+          score: null,
+        },
+      ],
+    },
+  },
+  {
+    kind: "takeSurvey",
+    name: "5. Post-discussion leadership survey",
+    config: {
+      questions: [
+        {
+          kind: "ScaleQuestion",
+          id: "4",
+          questionText:
+            "Rate the how much you would like to be the group leader.",
+          lowerBound:
+            "I would most definitely not like to be the leader (0/10)",
+          upperBound: "I will fight to be the leader (10/10)",
+          score: null,
+        },
+      ],
+    },
+  },
+  {
+    kind: "voteForLeader",
+    name: "6. Vote for the leader",
+    config: {
+      votes: {
+        "participant-0": "not-rated",
+        "participant-1": "not-rated",
+        "participant-2": "not-rated",
+      },
+    },
+  },
+  {
+    kind: "takeSurvey",
+    name: "7. Post-discussion work",
+    config: {
+      questions: [
+        {
+          kind: "RatingQuestion",
+          id: "5",
+          questionText:
+            "Please rating the following accoring to which is best for survival",
+          item1: {
+            name: "compas",
+            imageUrl:
+              "https://m.media-amazon.com/images/I/81NUeKWdiQL._AC_UF1000,1000_QL80_.jpg",
+          },
+          item2: {
+            name: "blanket",
+            imageUrl:
+              "https://img.freepik.com/free-psd/blanket-isolated-transparent-background_191095-10098.jpg?size=338&ext=jpg&ga=GA1.1.1700460183.1712448000&semt=sph",
+          },
+          choice: null,
+          confidence: null,
+        },
+      ],
+    },
+  },
+  {
+    kind: "leaderReveal",
+    name: "8. Leader reveal",
+    config: {
+      pendingVoteStageName: "6. Vote for the leader",
+      revealTimestamp: null,
+    },
+  },
+  {
+    kind: "takeSurvey",
+    name: "9. final satisfaction survey",
+    config: {
+      questions: [
+        {
+          kind: "ScaleQuestion",
+          id: "6",
+          questionText:
+            "Rate how happy you were with the final outcome.\nAlso indicate your overall feeling about the experience.",
+          lowerBound: "I was most definitely disappointed (0/10)",
+          upperBound: "I was very happy (10/10)",
+          score: null,
+        },
+      ],
+    },
+  },
+];
+
+const DEFAULT_STAGE_MAP = DEFAULT_STAGES.reduce((acc, stage) => {
+  acc[stage.name] = stage;
+  return acc;
+}, {} as Record<string, any>);
```

Comment:
Don't use `any` types - typescript has great typing support, aim to use it as much as you can. :) 

You can use `Object`, or `unknown`. Better to use "unknown" and then later to typecast to what you want it to be explitily, e.g. `... as Stage[]`. 

Also make sure to add the type for `DEFAULT_STAGE_MAP` too.